api/python/cellxgene_census/src/cellxgene_census/experimental/ml/pytorch.py

```diff
@@ -451,6 +475,9 @@ def __init__(
                 available for processing via the iterator. This allows network (or filesystem) requests to be made in
                 parallel with client-side processing of the SOMA data, potentially improving overall performance at the
                 cost of doubling memory utilization. Defaults to ``True``.
+            shuffle_chunk_count:
+                The number contiguous blocks (chunks) of rows to read at random and then concatenated and shuffled.
+                Larger number for `shuffle_chunk_count` correspond to more randomness in the shuffling.
```

Comment:
There are some typos here, but I think this could also be explained a little more clearly (though maybe the whole approach should be documented somewhere).

If we're okay with calling the intermediate collection a "shuffle buffer", then:

```suggestion
                The number of contiguous blocks (chunks) of rows sampled for each shuffle buffer.
                Each shuffle buffer is then shuffled again, and used a source for each batch.
                Larger numbers correspond to more randomness per training batch.
```

Otherwise:

```suggestion
                The number of contiguous blocks (chunks) of rows sampled to then concatenate and shuffle.
                Larger numbers correspond to more randomness per training batch.
```